src/main/java/seedu/address/logic/commands/schedule/FindCommonTimingCommand.java

```diff
@@ -55,51 +58,31 @@ public FindCommonTimingCommand(IsTagInPersonPredicate predicate, LocalDate date)
      * @param event event that needs to be accounted for
      */
     public void blockTimeSlots(int[] timeSlots, Event event) {
-        LocalTime startTime = event.getTime();
+        Event eventAtCurrentDate = event.getEventAtDate(date);
+        LocalTime startTime = eventAtCurrentDate.getTime();
         int hours = startTime.getHour();
         int minutes = startTime.getMinute();
         int startSlot = hours * 2;
         if (minutes >= 30) {
             startSlot += 1;
         }
-
-        LocalTime endTime = event.getEndTime();
+        LocalTime endTime = eventAtCurrentDate.getEndTime();
         int endHours = endTime.getHour();
         int endMinutes = endTime.getMinute();
         int endSlot = endHours * 2;
         if (endMinutes > 30) {
             endSlot += 1;
         }
```

Comment:
For this, you should also consider the timings that is less 30mins.
If the event ends at 10:15, then the person will only be free at 10:30
If the event ends at 10:45, then the person will only be free at 11:00

Reply:
Also the endSlot should be 48, if the endTime is 00:00. Currently, endSlot is 0 when endTime is 00:00

Reply:
Have amended this